src/docs/enriching-error-data/breadcrumbs.mdx

```diff
@@ -35,6 +35,3 @@ hint. The function can modify the breadcrumb or decide to discard it entirely:
 For information about what can be done with the hint see [Filtering Events](/error-reporting/configuration/filtering/#before-breadcrumb).
 
 ## Next Steps:
```

Comment:
```suggestion
```

Also remove the headline, right?

Reply:
Yes, and the redirects. Thanks for being a second set of eyes!